### PR TITLE
Managed Network Fabric extension release 6.1.0

### PR DESCRIPTION
**Pull request by aytripathi**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/managednetworkfabric/azext_managednetworkfabric/aaz/latest/networkfabric/tap/_resync.py [hunk 1]
```diff
@@ -45,8 +45,8 @@ def _build_arguments_schema(cls, *args, **kwargs):
         # define Arg Group ""
 
         _args_schema = cls._args_schema
-        _args_schema.network_tap_name = AAZStrArg(
-            options=["--network-tap-name"],
```
Comment (kairu-ms) [inline src/managednetworkfabric/azext_managednetworkfabric/aaz/latest/networkfabric/tap/_resync.py:49]: It seems the argument option name is changed. Is it by design?

File: src/managednetworkfabric/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+6.1.0
```
Comment (AllyW) [inline src/managednetworkfabric/HISTORY.rst:5]: Backward incompatible changes should increase version in major part, which is 7.0.0
please refer to extension versioning doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md

Comment (aytripathi) [reply src/managednetworkfabric/HISTORY.rst:5]: We are wanting to keep the az CLI release version in correspondence to the latest release version of our service managed network fabric

**Review by Moazzem-Hossain-pixel — COMMENTED** [review]
Thanks for support.

**Review by Moazzem-Hossain-pixel — COMMENTED** [review]
Thank you so much

**Pull request merged**